docs/configuration.md

```diff
@@ -210,3 +210,21 @@ REMS can be configured to delete applications after a set period of time has pas
 ## Shopping cart
 
 REMS has a shopping cart feature which allows bundling multiple resources into single application. Shopping cart is enabled by default, and it can be enabled or disabled using the `:enable-cart` key in your `config.edn`. See [config-defaults.edn](https://github.com/CSCfi/rems/blob/master/resources/config-defaults.edn) for details.
+
+## Attachment Malware Scanning
+
+REMS can be configured to automatically scan user uploaded files for malware. 
+
+```clojure
+ :malware-scanning {:scanner-path "/path/to/malware/scanner" ; Path to scanner executable
+                    :logging true} ; Whether REMS should log STDERR of malware scanner 
+```
+
+The executable must implement the following behaviour:
+
+> Given a binary stream on `STDIN`, exit with a status-code of zero if and only if the file constituting the binary stream is scanned to be malware free.
+
+In its current implementation, users will be notified with a generic error message if the file they attach does not pass the malware scanner.
+
+When logging is enabled REMS will output any message that the malware scanner executable writes to `STDOUT` at `INFO` level, 
+and anything written to `STDERR` at `ERROR` level.
```

Comment:
We could add here also a mention here how to get ClamAV working and/or that we have tested using it.